Add notes for Chocolatey Central Management 0.12.0 (#900)
chocolatey/docs: input/en-us/central-management/release-notes.md

- Please see our [GitHub repository issues](https://github.com/chocolatey/chocolatey-licensed-issues/labels/CentralManagement).

## 0.12.0 (November 29, 2023)

### Features

- Provide ability to create a Deployment Plan for a single outdated package on all affected Computers.
- Provide ability to create a Deployment Plan for all outdated packages on all Computers.
- Provide ability to create an empty Deployment Plan directly for a specific Computer - see [Licensed #259](https://github.com/chocolatey/chocolatey-licensed-issues/issues/259).
- Provide ability to create a Deployment Plan for all outdated packages for a single Computer - see [Licensed #281](https://github.com/chocolatey/chocolatey-licensed-issues/issues/281).
- Provide ability to duplicate an existing Group.
- Provide ability to create an empty Deployment Plan directly for a specific Group.

### Bug Fixes

> - Fix - Certain roles within the Chocolatey Central Management website are able to impersonate other users using the "Login as this user" action menu - see [Licensed #318](https://github.com/chocolatey/chocolatey-licensed-issues/issues/318).

Reviewer: This doesn't explain what the fix is. It's a statement of functionality. Can we reword this?

- Fix - Unnecessary permissions should not be present in the `Edit Roles` and `Permissions` modal windows.

> - Fix - Notification about scheduled Deployment Plan failing to start uses incorrect date format.

Reviewer: ```suggestion
- Fix - Notification about a scheduled Deployment Plan failing to start uses incorrect date format.
```

- Fix - Inconsistent presentation of Export and Action buttons across pages.
- Fix - Missing TimeZone setting for users in Colombia - (UTC - 05:00) Bogota, Lima, Quito, Rio Branco - see [Licensed #356](https://github.com/chocolatey/chocolatey-licensed-issues/issues/356).
- Fix - The HTTP error page of the Chocolatey Central Management Website does not use standard styling.

> - Fix - Different colors are used for the "yes/no" badges on various tables across the website.

Reviewer: ```suggestion
- Fix - Ensure colors for the "yes/no" badges are consistent on various tables across the website.
```

- Fix - Encryption Settings information callout is dismissible when it shouldn't be.
- Fix - Password reset link no longer displays an "invalid link" message after updating the Encryption Passphrase.

> - Fix - Tables of data do not retain current page when refreshed automatically.

Reviewer: I don't know what this is when reading it. Can it be reworded?

- Fix - The casing of the word for product "Excel" is incorrect on all export buttons.
- Fix - Names of Deployment Steps can cause an overflow on some modal windows.
- Fix - Action buttons on tables that only contain one option are hidden.
- Fix - Existing configuration stored in the `appsettings.json` file are not respected when upgrading either the Chocolatey Central Management Database or Service packages.
- Fix - Elements on page overlap when a Passphrase is required to be reset by an Admin.

### Improvements

- Provide ability to have a Group Details screen.
- Provide ability to automatically fetch updated information for main Deployment Plan page.

> - Provide ability to see information about the last refreshed time for KPI Dashboard.

Reviewer: ```suggestion
- Provide ability to see information about the last refreshed time for Dashboard.
```

> - Provide ability to see the number of Computers that have a particular piece of Software installed - see [Licensed #231](https://github.com/chocolatey/chocolatey-licensed-issues/issues/231).

Reviewer: ```suggestion
- Provide ability to see the number of Computers that have specific Software installed - see [Licensed #231](https://github.com/chocolatey/chocolatey-licensed-issues/issues/231).
```

- Provide ability to duplicate a Deployment Step within a Deployment Plan - see [Licensed #332](https://github.com/chocolatey/chocolatey-licensed-issues/issues/332).
- Information about the number of ready/active Deployment Plans is available within the Dashboard page.
- Reporting - Truncate all Excel sheet names to required 31-char limit.

> - Confusing TimeZone options for Central Europe in Chocolatey Central Management - see [Licensed #325](https://github.com/chocolatey/chocolatey-licensed-issues/issues/325).

Reviewer: This doesn't fatty what's been done. It's just a statement. Can it be reworded?

> - Add a KPI to the Dashboard page to show the number of Computers that have not recently reported into Chocolatey Central Management.

Reviewer: ```suggestion
- Add an item to the Dashboard page to show the number of Computers that have not recently reported into Chocolatey Central Management.
```

- Shorten generated names for Deployment Steps to remove redundant "Deployment Step" prefix.

### Documentation

> - Normalize wording for Deployment Plan and Deployment Step in UI

Reviewer: ```suggestion
- Normalize wording for Deployment Plan and Deployment Step in UI.
```
